ingestion/src/metadata/ingestion/source/dashboard/superset/api_source.py

```diff
@@ -76,7 +76,15 @@ def yield_dashboard(
             description="",
             dashboardUrl=dashboard_details["url"],
             owner=self.get_owner_details(dashboard_details),
-            charts=[chart.fullyQualifiedName.__root__ for chart in self.context.charts],
+            charts=[
+                fqn.build(
+                    self.metadata,
+                    entity_type=Chart,
+                    service_name=self.context.dashboard_service.fullyQualifiedName.__root__,
+                    chart_name=chart.name.__root__,
+                )
+                for chart in self.context.charts
```

Comment:
@OnkarVO7, it might be better to setup the fqn in the charts and use them directly instead of building fqn every time?

Reply:
@sureshms we will be handling that in a separate [issue](https://github.com/open-metadata/OpenMetadata/issues/10133).
This issue will remove the entire `entity_ref` from ingestion topology and use the fqn everywhere.